fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1693,60 +1713,82 @@ protected synchronized void runOneCycle() {
         listener.setMetaContext(metaContext);
     }
 
-    public synchronized boolean replayJournal(long toJournalId) throws JournalException {
-        long newToJournalId = toJournalId;
-        if (newToJournalId == -1) {
-            newToJournalId = getMaxJournalId();
-        }
-        if (newToJournalId <= replayedJournalId.get()) {
-            return false;
-        }
-
-        LOG.info("replayed journal id is {}, replay to journal id is {}", replayedJournalId, newToJournalId);
+    /**
+     * Replay journal from replayedJournalId + 1 to toJournalId
+     * used by checkpointer/replay after state change
+     */
+    public boolean replayJournal(long toJournalId) throws JournalException {
+        LOG.info("replayed journal id is {}, replay to journal id is {}", replayedJournalId, toJournalId);
```

Comment:
LOG.info("start to replay journal from {} to {}", replayedJournalId + 1, toJournalId)

Reply:
Done